blackbelt/deployment.py

```diff
@@ -5,7 +5,7 @@
 
 
 def deploy_staging():
-    branch_name = get_current_branch()
+    branch_name = str(get_current_branch())
```

Comment:
I guess it would be better if `get_current_branch()` retutns strings and not bytes :)

Reply:
It would but this is a hotfix so PRs welcome to support py3 properly :)